### PR TITLE
Update `scripts/index` and gothic sourcer article

### PR DESCRIPTION
**Pull request by muczc1wek**

### CODE REVIEW

**Comment by kamilkrzyskow** [comment]
The language server is AFAIK tool agnostic, it's not bound to VS Code, but the extension using the language server is.
I also think if there are alternative tools to some aspects of Gothic Sourcer, then the article should have an opinionated comment about this, but I don't know about alternative 😆

**Comment by muczc1wek** [comment]
> The language server is AFAIK tool agnostic, it's not bound to VS Code, but the extension using the language server is.


So how should I put this? I know dls != vscode-daedalus but for the regular user these things are the same

**Comment by kamilkrzyskow** [comment]
Perhaps they're seen as the same, but docs should strive for factuality. Of course there are some opinionated takes here or there, but there should be a good reason for those.

Here instead of "Also known as Daedalus Language Server", for example change it to "The VS Code extension acts as a proxy for the [Daedalus Language Server](https://github.com) to provide [list of features]" or "The VS Code extension provides [list of features] and uses [Daedalus Language Server](https://github.com) as the back-end for them." etc. maybe ChatGPT has something better.

File: docs/zengin/scripts/index.md [hunk 1]
```diff
@@ -4,7 +4,16 @@ ZenGin uses its own scripting language called Daedalus. It is similar to C progr
...
+
+### Gothic Sourcer
+Gothic Sourcer is a tool that allows you to edit Daedalus scripts with syntax highlighting and built-in compiler. It also has a feature to decompile scripts from the game. However, it lacks support for script packages like [Ikarus](extenders/ikarus/index.md) and [LeGo](extenders/lego/index.md).
+
+[:octicons-arrow-right-24: Read More](../tools/gothic_sourcer.md)
+
+### Daedalus VS Code extension
+he VS Code extension provides syntax highlighting, code completion, and other features for Daedalus scripts, and uses [Daedalus Language Server](https://github.com/Kirides/DaedalusLanguageServer) as the back-end for them.
+
+[:octicons-arrow-right-24: Read More](../tools/daedalus_tools/daedalus_language_server.md)
```
Comment (kamilkrzyskow) [inline docs/zengin/scripts/index.md:18]: The section title `###` mentions VS Code Extension, but the link for Read More is for the Language Server 🤔 
So maybe re-structure the paragraph to say "The Daedalus Language Server provides syntax [...], and it is used by the VS Code extension."

But now this "short" excerpt is more explicit than the actual `daedalus_language_server.md` file 😆 
So it's hard to say how to structure it properly

**Review by kamilkrzyskow — CHANGES_REQUESTED** [review]

File: docs/zengin/tools/gothic_sourcer.md [hunk 2]
```diff
@@ -1,4 +1,7 @@
 # Gothic Sourcer
-Gothic Sourcer can be used to do a lot of things.
-!!! todo
-    TODO
+Gothic Sourcer is a multipurpose tool. It can be used to edit Daedalus scripts, decompile game animations, and more. 
+
+## Download
+You can download the latest version of Gothic Sourcer from the [Resource Manager](https://worldofplayers.ru/threads/41942/).
```
[the remark this answers is not in the document]
Comment (muczc1wek) [reply docs/zengin/tools/gothic_sourcer.md:5]: I think there is an installer on gratt's gdrive

File: docs/zengin/scripts/index.md (hunk 1 above)
Comment (muczc1wek) [reply docs/zengin/scripts/index.md:18]: The dls article is mostly about vscode-daedalus. I don't see a reason to not link to it in "read more"

Comment (kamilkrzyskow) [reply docs/zengin/scripts/index.md:18]: OK, then perhaps there is no need for the DLS link to GitHub, as it's in the Read More article, or add the extension link here as well to make it more synced.

File: docs/zengin/tools/daedalus_tools/daedalus_language_server.md [hunk 3]
```diff
@@ -1,5 +1,5 @@
 # Daedalus Language Server
-[Daedalus Language Server](https://github.com/Kirides/DaedalusLanguageServer) is a language server for Gothic's scripting language - Daedalus, created by [Kirides](https://github.com/kirides). However, it is mostly known as VSCode extension, that is in fact named [vscode-daedalus](https://github.com/kirides/vscode-daedalus).
+[Daedalus Language Server](https://github.com/Kirides/DaedalusLanguageServer) is a language server for Gothic's scripting language - Daedalus, created by [Kirides](https://github.com/kirides). However, it is mostly known for a VSCode extension - [vscode-daedalus](https://github.com/kirides/vscode-daedalus).
```
[the remark this answers is not in the document]
Comment (muczc1wek) [reply docs/zengin/tools/daedalus_tools/daedalus_language_server.md:2]: I will leave it as it is

[the remark this answers is not in the document]
Comment (kamilkrzyskow) [reply docs/zengin/tools/daedalus_tools/daedalus_language_server.md:2]: Fine. Still, there should be a clear separation between the extension and the DLS imo, but this section can be improved later when more text editors incorporate the Language Server etc.

**Review by kamilkrzyskow — APPROVED** [review]

**Comment by kamilkrzyskow** [comment]
LGTM, nice work on the preservation of the Gothic Sourcer in a separate repository 👍 
You can squash merge when you want :v:

**Pull request merged**